easybuild/easyconfigs/m/Mesa/Mesa-10.4.5-intel-2015a-Python-2.7.9.eb

```diff
@@ -59,7 +59,7 @@ configopts += " --disable-driglx-direct --with-gallium-drivers='' --disable-egl"
 # package-config files for os dependencies are in an os specific place
 #preconfigopts = ' PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/lib64/pkgconfig/:/usr/share/pkgconfig" '
 
-prebuildopts = 'CPATH="$EBROOTLIBDRM/include/libdrm" '
+prebuildopts = 'env CPATH="$EBROOTLIBDRM/include/libdrm" '
```

Comment:
eek, I don't think we should be hard redefining `$CPATH` and totally ignore the current value here...

@riccardomurri fixed with https://github.com/riccardomurri/easybuild-easyconfigs/pull/5